api/src/controllers/application.js

```diff
@@ -302,7 +302,12 @@ router.post("/notify/docs-military-preparation/:template", passport.authenticate
   }
 
   const mail = await sendTemplate(parseInt(template), {
-    emailTo: [{ name: `${toReferent.firstName} ${toReferent.lastName}`, email: toReferent.email }],
+    emailTo: toReferent.map((referent) => {
+      return {
+        name: `${referent.firstName} ${referent.lastName}`,
+        email: referent.email,
+      };
+    }),
```

Comment:
j'aime bien mettre au pluriel si il ya plusieurs referents -> `toReferents`
et le map, quand il n y a pas vraiment de triatement, j'essaie de le faire sans return aussi
```suggestion
    emailTo: toReferents.map((referent) => (
      {
        name: `${referent.firstName} ${referent.lastName}`,
        email: referent.email,
      })
    ),
```